packages/connection-encrypter-tls/test/index.spec.ts

```diff
@@ -46,8 +47,8 @@ describe('tls', () => {
     })
 
     await Promise.all([
-      encrypter.secureInbound(remotePeer, inbound),
-      encrypter.secureOutbound(localPeer, outbound, wrongPeer)
+      encrypter.secureInbound(inbound, remotePeer),
+      encrypter.secureOutbound(outbound, wrongPeer)
```

Comment:
Curious, do we need an `encrypterRemote` here like we have in [packages/connection-encrypter-plaintext/test/index.spec.ts](https://github.com/libp2p/js-libp2p/pull/2304/files#diff-b48cf156af4dc44ee636b9b7f7f19255ef04ce7ceb471c7e7c68dfb8d88ef6ae) ?

If we can do either/or, it would be nice to use the same setup to not cause confusion and questions like this one :)

If we can't, maybe a comment explaining why? (maybe only for libp2p noobs like myself)

Reply:
It would make the test easier to read but this encrypter is stateless which is why it works as-is.